lib/chat/server/messages/reward-callback.js

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * Reldens - RewardCallback
+ *
+ */
+
+const { Logger } = require('@reldens/utils');
+const { ChatConst } = require('../../constants');
+
+class RewardCallback
+{
+
+    static async sendMessage(props)
+    {
+        let { itemRewards, player_id, room_id, client, chatManager } = props;
+        if (null === client) {
+            return false;
+        }
+        let sendMessage = '';
+        for (let i = 0; i < itemRewards.length; i++) {
+            let { reward, item } = itemRewards[i];
+            let message = config.getWithoutLogs(
+                'client/rewards/titles/rewardMessage',
+                ChatConst.MESSAGES.REWARD
+            ).replace('%dropQuantity', reward.drop_quantity).replace('%itemLabel', item.label);
+            if (itemRewards.length > 1 && i !== itemRewards.length - 1) {
+                message += ' - ';
+            }
+            sendMessage += message;
+        }
+        client.send({
+            act: ChatConst.CHAT_ACTION,
+            f: 'Rewards',
```

Comment:
I took this example from another part of your code... honestly I don't really know what "f", "m" and "t" are. let's check it out on the next VC, because this should be changed on every other chat message, right?